LetsDinnerV2 MessagesExtension/View/CustomUIView/SectionSelectionInput.swift

```diff
@@ -79,8 +80,12 @@ class SectionSelectionInput : UIView {
         // Maybe adding constraints in layoutSubviews fix the bug of the toolnar not always appearing
         addConstraints()
         
-        // Control the selected bubble (First time)
-        sectionsCollectionView.selectItem(at: [0,0], animated: true, scrollPosition: .top)
+        if viewFirstInit {
+            // Show the selected bubble (First time)
+            sectionsCollectionView.selectItem(at: [0,0], animated: true, scrollPosition: .top)
+            viewFirstInit = false
```

Comment:
Really? There is no other way? That looks very dodgy

Reply:
Actually the bug does not seem to be there on 1.0.25 so no need for this

Reply:
the bug is: when hiding the keyboard, if the selectedSection are saved at the second item, but the bubble appears at Misc when showing the keyboard again